docs/source/notes/mkldnn.rst

```diff
@@ -26,7 +26,7 @@ Users can disable MKLDNN backend by:
 Bfloat16 (BF16) on MKLDNN backend
 ---------------------------------------------------
 
-Starting in PyTorch 2.4, there is a set of APIs to control the internal computation precision
+Starting in PyTorch 2.9, there is a set of APIs to control the internal computation precision
```

Comment:
Was 2.4 wrong?

Reply:
No, the APIs are finally upstreamed in 2.9, 2.4 didn't have that. https://github.com/pytorch/pytorch/pull/125888 was firsted drafted in PyTorch 2.4 window, but is upstreamed in 2.9. `Mkldnn Docs` needs to be updated to 2.9.